fastlane/Fastfile

```diff
@@ -146,6 +136,12 @@ platform :android do
     end
 
     lane :google_play_distribute do |options|
+        version_number_string = options[:version_number_string]
+        setup_ci(force: true)
+        update_version_number version_number_string:version_number_string
+        tag = "#{version_number_string}-android"
```

Comment:
Just curious, Why a separate tag for android and iOS? Aren't both releases done on CI?

Reply:
Just how we've always done it but I agree it's something I'd like to look at in a follow up